aries-site/src/pages/components/dateinput.mdx

```diff
@@ -10,7 +10,7 @@ import {
   componentName="DateInput"
   docs="https://v2.grommet.io/dateinput?theme=hpe"
   code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/dateinput/DateInputSimpleExample.js"
-  figma="https://www.figma.com/file/Mf9if6zyGETTWW5b9PsYNQ/HPE-Date-Input-Component?node-id=1%3A2"
+  figma="https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?type=design&node-id=444-9254&t=KjhQZHbAZR51Nmwu-11"
```

Comment:
This one is linking to the frame with the component, but all the other PRs link to the frame with the Changelog -- do we want to be consistent?

Reply:
I guess its the wrong link we need to replace it with 
'https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?type=design&node-id=1340-29596&t=bDcQLYlYOWfjL0e5-11'
we need to raise new PR or you can edit and update?

Reply:
You should be able to edit the file from this same PR:

https://github.com/grommet/hpe-design-system/assets/12522275/8c8dee37-a70a-4040-9e73-4c388a1f65e1